yt_dlp/extractor/tv2hu.py

```diff
@@ -2,61 +2,117 @@
 from __future__ import unicode_literals
 
 from .common import InfoExtractor
-from ..utils import int_or_none
+from ..utils import (
+    ExtractorError,
+    traverse_obj,
+)
 
 
 class TV2HuIE(InfoExtractor):
-    IE_NAME = 'tv2.hu'
-    _VALID_URL = r'https?://(?:www\.)?tv2\.hu/(?:[^/]+/)+(?P<id>\d+)_[^/?#]+?\.html'
+    IE_NAME = 'tv2play.hu'
+    _VALID_URL = r'(?!.*szalag/)https?://(?:www\.)?tv2play\.hu/(?P<id>[^#&?]+)'
```

Comment:
why `(?!.*szalag/)`?

Reply:
it means `tape` or something in hungarian
the site uses it to denote group of videos 
A show can have 2-3 groups with an id and a url in the form 
`tv2play.hu/szalag/<id>`
without explicitly excluding this, it will match this and result in error at first json download

Reply:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?tv2play\.hu/(?!szalag/)(?P<id>[^#&?]+)'
```